src/tokens/colors.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  white: '#FFF',
+  black: '#212123',
+  blue: {
+    1: '#edf4f7',
+    2: '#a9d6ea',
+    bright: '#4f66ee',
```

Comment:
have we talked to design team about renaming these to keep consistent with the numeric labeling (like below `4,3,2,1`)? feels like the right thing to do, but maybe there's a good reason for these adjectives `bright` and `dark`